static/locales/en-US.yaml

```diff
@@ -768,6 +770,8 @@ Tooltips:
     Fetch Feeds from RSS: When enabled, FreeTube will use RSS instead of its default
       method for grabbing your subscription feed. RSS is faster and prevents IP blocking,
       but doesn't provide certain information like video duration or live status
+    Fetch Automatically: When enabled, FreeTube will automatically fetch
+      your subscription feed on start and when switching profile.
```

Comment:
Not sure if important to mention but feed will also be fetched when opening a new window/tab

Reply:
The visual space between them can be fixed by adding the `:compact="true"` attribute to all 3 switches.

Reply:
> Not sure if important to mention but feed will also be fetched when opening a new window/tab

I don't think _"on start"_ is the best wording.
I'll see if I can think of something better or if there's any suggestions.
Maybe _"when a window is opened"_ or something.

Reply:
> The visual space between them can be fixed by adding the `:compact="true"` attribute to all 3 switches.

Thanks, I'll add that.

Reply:
I've made the switches compact and reworded the tooltip.